src/plugins/ntf_json.c

```diff
@@ -456,6 +456,10 @@ srpntf_json_store(const struct lys_module *mod, const struct lyd_node *notif, co
 
             /* we are done */
             goto cleanup;
+        } else if (notif_ts->tv_sec == from_ts && notif_ts->tv_sec == to_ts) {
+            srplg_log_errinfo(&err_info, srpntf_name, NULL, SR_ERR_INTERNAL,
+                "Not enough file space.");
```

Comment:
Please explain the exact problem in the message. Such as `Notification file <from>-<to> exceeded the maximum size <max-size> KB.`.